src/Core/MainMenu.cpp

```diff
@@ -0,0 +1,127 @@
+#include "MainMenu.h"
+#include"imgui.h"
+#include"backends/imgui_impl_glfw.h"
+#include"backends/imgui_impl_opengl3.h"
+
+#include<iostream>
+#include<glad/glad.h>
+#include<GLFW/glfw3.h>
+
+
+void MainMenu::DrawCenteredText(const std::string text)
+{
+	const auto windowWidth = ImGui::GetWindowSize().x;
+	const auto textWidth = ImGui::CalcTextSize(text.c_str()).x;
+
+	ImGui::SetCursorPosX((windowWidth - textWidth) * 0.5f);
+	ImGui::Text(text.c_str());
+}
+
+bool MainMenu::DrawWindowSizedButton(const std::string text)
+{
+
+	const auto windowWidth = ImGui::GetWindowSize().x;
+	if (ImGui::Button(text.c_str(), ImVec2(windowWidth -15, 50)))
+	{
+		return true;
+	}
+	return false;
+}
+
+void MainMenu::InitializeGlfw()
+{
+	glfwWindowHint(GLFW_CONTEXT_VERSION_MAJOR, 3);
+	glfwWindowHint(GLFW_CONTEXT_VERSION_MINOR, 3);
+	glfwWindowHint(GLFW_OPENGL_PROFILE, GLFW_OPENGL_CORE_PROFILE); //(core profile (use only modern functions to hell with old ones))
+}
+
+
+void MainMenu::Draw()
+{
+	//initialize GLFW
+	glfwInit();
+
+	//hints (information about version to glfw) //version 3.3
+	InitializeGlfw();
+
+	GLFWwindow* window = glfwCreateWindow(1600, 800, "Main menu", /*glfwGetPrimaryMonitor()*/  nullptr, nullptr); //width, height, name, FULLSCREAN(nullptr/?), "not important"
```

Comment:
1600 and 800 are magic numbers. Please add constant variables like or even better - a struct like
```
struct window {
    unsigned width = 1280;
    unsigned height = 720; 
};
```

Then use these variables here.

Reply:
Also these comments worsen code readability

Reply:
Won't fix